misc/extract-schema.pl

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env perl
+use 5.26.1;
+use feature 'signatures';
+no warnings 'experimental::signatures';
+
+use Getopt::Long;
+use YAML::XS qw(LoadFile);
+use Mojo::JSON qw(encode_json);
+use Mojo::File qw(path);
+use Data::Visitor::Tiny qw(visit);
+
+my $schema_file = "$FindBin::Bin/../json-schema/input.yaml";
```

Comment:
DirSelf is available in cpanfile already and provides a less wtf way of getting this information.

Reply:
I've changed it but I'm not 100% sold that it's _really_ that much less of a wtf.